PWGHF/D2H/TableProducer/candidateSelectorBplusToD0PiReduced.cxx

```diff
@@ -89,10 +91,10 @@ struct HfCandidateSelectorBplusToD0PiReduced {
     }
   }
 
-  void process(HfCandBplus const& hfCandBs,
-               HfCand2ProngReduced const&,
-               HfTracksPidReduced const&,
-               HfCandBpConfig const& configs)
+  void process(HfRedCandBplus const& hfCandBs,
+               HfRed2Prongs const&,
```

Comment:
Indeed, we don't subscribe to the equivalent table for B0. We don't need it anymore.

Reply:
thanks @AlexBigO for the confirmation. Then please remove it in your next commit.

Reply:
I removed it (see [here](https://github.com/AliceO2Group/O2Physics/blob/05c010e7e2a3ccf61ee3ed4c2ed09fb554e8e8bf/PWGHF/D2H/TableProducer/candidateSelectorBplusToD0PiReduced.cxx#L94)) but it does not appear on this review comment (I guess it is because this comment is from a previous version of the branch).

Reply:
sorry I overlooked it. Fine for me then! Thanks a lot.